sdk/translation/azure-ai-documenttranslator/pom.xml

```diff
@@ -35,6 +35,9 @@
     <javaModulesSurefireArgLine>
       --add-exports com.azure.core/com.azure.core.implementation.jackson=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <codesnippet.skip>false</codesnippet.skip>
+    <javadocDoclet></javadocDoclet>
+    <javadocDocletOptions></javadocDocletOptions>
```

Comment:
Also update the README.md file to use the new code snippets tool and remove `embedme`.